about.html

```diff
@@ -0,0 +1,109 @@
+        </div>
+        <div class="nav-wrapper-desktop">
+            <h1>IOT-Horizons</h1>
+            <div>
+                <a href="index.html">Home</a>
+                <a href="">Program</a>
+                <a href="">Join</a>
+                <a href="">Sponsor</a>
+                <a href="">News</a>
+                <a href="">IOT Campaign</a>
+            </div>
+        </div>
+        <div class="presentation-container-about">
+            <header>
+                <h3>"Hello Tech world"</h3>
+                <h1>IOT CONGRESS 2023</h1>
+            </header>
+            <h6 class="line">────</h6>
+            <p class="main-description-about">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sed, facere? Non blanditiis eligendi eaque adipisci consectetur expedita, nobis quasi tenetur dolore, officia quas fuga ratione debitis dolores, molestias possimus illum.</p>
+            <div class="presentation-container-footer">
+                <p>Please contact us per Email for any further questions about Global Congress 2022!</p>
+                <h4>iot@horizons.org</h4>
+            </div>
+        </div>    
+    </section>
+    <section class="creative-commons">
+        <h2>IOT-HOrizons 2023 Logo</h2>
+        <h6 class="line">────</h6>
+        <p>The logo of iOT Horizons 2023 was decided through the logo competition on 23 January</p>              
+        <h1>IOT-Horizons</h1>
+    </section>
+    <section class="see-past">
+        <h2>See the past IOT Horizons</h2>
+        <h6 class="line">────</h6>
+        <p class="text-past">Take a look at the last two IOT Horizons which took place in Abuja and Lagos</p>
+        <div class="meet-container">
+            <article class="first-meet">
+                <div class="last-meetings">
+                    <h3>2021</h3>
+                    <p>BC Global Summit 2021 in Abuja</p>
+                </div>
+            </article>
+            <article class="second-meet">
+                <div class="last-meetings">
+                    <h3>2022</h3>
+                    <p>BC Global Summit 2019 in Lagos</p>
+                </div>
+            </article>
+        </div>
+    </section>
+    <section class="partner-setup">
+        <h2>Partner</h2>
+        <ul class="sponsors">
+            <li class="mozilla">mozilla</li>
+            <li class="google">Google</li>
+            <li class="naver">Goal</li>
+            <li class="daumkakao">IOT-technology</li>
+            <li class="airbnb"><i class="fa-solid fa-dna"></i>Prime</li>
+        </ul>
+        <footer>
+            <div class="image-footer">
+                <h1>IOT CONGRESS 2023</h1>
+            </div>
+            <div class="footer-information">
+                <h3>2023 IOT Bootcamp Course Nigeria. Some Rights Reserved</h3>
+                <p>Unless otherwise specified, all content on the website is available under a Creative Commons Attribution 2.0 Nigeria license.</p>
+            </div>
```

Comment:
- [ ] Great job working on the footer but I think it would be better if you make the font size smaller in the mobile version and add the space between them to match the given design as much as possible since it is very important to stick with the design 💯🚀🤩🙏🏻🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/219304840-1395f02e-89d3-4f2e-bad1-38e6a1e76f9a.png)


- Given design
![image](https://user-images.githubusercontent.com/58553711/219304915-efac3b30-6a85-4115-bacf-ffdf75e41b89.png)